content/04.results.md

```diff
@@ -43,11 +43,28 @@ This pattern continues filling out the rest of the grid.
 The last most column consists of pooling every relationship specific label function and proceeding as above.
 ](https://raw.githubusercontent.com/greenelab/snorkeling/master/figures/label_sampling_experiment/transfer_test_set_auprc.png){#fig:gen_model_auprc}
 
+### Random Label Function Gen Model Analysis
+![
+A grid of area under the receiver operating curve (AUROC) for each edge type.
+Each plot consists of adding a single label function on top of the baseline model.
+This label function emits a positive (top row) or negative (bottom row) label at specified frequencies.
+The error bars represents confidence intervals for emission frequency.
```

Comment:
How big of confidence intervals? 90% 95%? Something else?

Reply:
95% will add to both sentences.

Reply:
ok 👍 

Confidence intervals also look like they relate to what's on the Y axis (instead of emission freq which is X-axis). After changing these things. Feel free to merge.